File: level 105/homework/task 01/homework1.py
def func(array):
    result = 0
    for i in range(len(array)):
        for x in range(len(array[i])):
            result += array[i][x]
    return result

print(func([
    [1,3],
    [1,4],
    [4,1],
    [2,2]
]))

File: level 063/homework/task 01/homework1.js
// https://www.codewars.com/kata/53dbd5315a3c69eed20002dd/train/python

function filter_list(l) {
    result = []
    for (let i of l){
      if (typeof i === "number"){
        result.push(i)
      }
    }
    return result
}